### PR TITLE
Make the documents headings clearer.

### PR DESCRIPTION
**Pull request by aakankshaduggal**
Making some document headings clearer as they were not rendering properly earlier in JB

### CODE REVIEW

**Review by oindrillac — APPROVED** [review]
/lgtm

File: docs/project_structure/project-structure.md [hunk 1]
```diff
@@ -19,9 +19,10 @@ Our approach to overcoming this issue is simply to focus on the fewest number of
 * Slack
 	* Project channel(s) ([example](https://join.slack.com/t/operatefirst/shared_invite/zt-o2gn4wn8-O39g7sthTAuPCvaCNRnLww))
 </br></br>
-# What are the phases of a data science project?:
 
-### **1. Project Definition and Use Case Understanding**:
+# What are the phases of a data science project?
+
+### 1. Project Definition and Use Case Understanding:
```
Comment (suppathak) [inline docs/project_structure/project-structure.md:25]:
```suggestion
### 1. Project definition and use case understanding:
```

File: docs/develop_collaborate/meteorize.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
- **How to Meteorize the repository**
-    - To meteorize your github repository, first start with making a jupyterbook. You can refer to the tutorial [here.](https://github.com/AICoE/meteor/blob/main/docs/build-jupyterbooks.md) 
-    - After you have the jupyterbook ready, head to [meteor](https://shower.meteor.zone/) to meteorize the project repository.
\ No newline at end of file
+ # How to Meteorize the repository
```
Comment (suppathak) [inline docs/develop_collaborate/meteorize.md:1]:
```suggestion
 # How to meteorize the repository
```

File: docs/project_structure/project-structure.md [hunk 3]
```diff
@@ -37,7 +38,7 @@ The project outline can include:
  
 Once this is done, you can move on to step 2, OR share your new project proposal with the Operate Fist DS Community on [Slack](https://join.slack.com/t/operatefirst/shared_invite/zt-o2gn4wn8-O39g7sthTAuPCvaCNRnLww) (#data-science) to get some feedback and potential contributors.   
 
-### **2.** **Research and Problem Understanding**:
+### 2. Research and Problem Understanding:
```
Comment (suppathak) [inline docs/project_structure/project-structure.md:41]:
```suggestion
### 2. Research and problem understanding:
```

File: docs/project_structure/project-structure.md [hunk 4]
```diff
@@ -47,7 +48,7 @@ Now that you have a problem you'd like to solve, its time to do some research!!!
 
 3. Add a `research.md` doc to your repo outlining the papers and projects explored above. This may be useful to you for reference or documentation later on. Plus, anyone who needs to do the same research in the future will thank you. 
 
-### **3.** **Data Preparation and EDA**: 
+### 3. Data Preparation and EDA:
```
Comment (suppathak) [inline docs/project_structure/project-structure.md:51]:
```suggestion
### 3. Data preparation and EDA: 
```

File: docs/serving_monitoring/monitor-jh.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-* **How to Monitor JupyterHub workloads using Grafana dashboards** 
+# How to Monitor JupyterHub workloads using Grafana dashboards
```
Comment (suppathak) [inline docs/serving_monitoring/monitor-jh.md:1]:
```suggestion
# How to monitor JupyterHub workloads using Grafana dashboards?
```

**Comment by suppathak** [comment]
Oops @aakankshaduggal , I just made some comments on uppercase and lowercase, :) .

**Comment by aakankshaduggal** [comment]
> Oops @aakankshaduggal , I just made some comments on uppercase and lowercase, :) . Although its not high priority. Lmk if it makes sense.

@suppathak - Thanks for your suggestions, I will address these comments in a separate PR :)

**Pull request merged**